errext/print.go

```diff
@@ -0,0 +1,30 @@
+package errext
+
+import (
+	"errors"
+
+	"github.com/sirupsen/logrus"
```

Comment:
It might be good idea to keep this package free of external dependencies, in such case I'm completely open to suggestions for a new location for this function.

Reply:
Yeah but as explained  in https://github.com/grafana/k6/issues/2958 it might not be that easy todo.

I feel like we either:
1. move this to some other package (`lib` is the usual junk drawer)
2. make the helper here return `hint`/`fields` (`map[string]string`) and the `errText`

Reply:
Good idea, let's go with opt. 2, [as you suggested](https://github.com/grafana/k6/pull/3416#discussion_r1370142485).